jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedGarbageCollectorMXBeanImpl.java

```diff
@@ -77,16 +80,30 @@ public GcInfo getLastGcInfo() {
 	 */
 	private native GcInfo getLastGcInfoImpl(int id);
 
-	static GcInfo buildGcInfo(long index, long startTime, long endTime,
+	static GcInfo buildGcInfo(long index, long startTime, long endTime, int[] ids,
```

Comment:
This seems wrong, passing the ids in every call, but only actually using them the first time.

Reply:
yes, you are right, we just use them once, will try to create a initial jni function to set names list instead of passing  ids for every time.

Reply:
I'll put this in draft mode until then.

Reply:
it is ready for re-review.